crates/torii/graphql/src/tests/metadata_test.rs

```diff
@@ -23,8 +23,8 @@ mod tests {
               content {
                 name
                 description
-                cover_uri
```

Comment:
Hi @broody , What about here? should we keep format casing as-is, for `cover_uri` and `icon_uri`? Like I did for `cover_img`, `icon_img`

Reply:
These r good to be converted to camelCase since metadata object is something we define. Same with the page info.